test/integration/.dockerignore

```diff
@@ -0,0 +1,22 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
This is a copy of file from elsewhere in this repo. The original copyright should be kept.